bindings/python/src/pgvecto_rs/psycopg/__init__.py

```diff
@@ -0,0 +1,48 @@
+from psycopg import Connection, ProgrammingError
+from psycopg.adapt import Loader, Dumper
+from psycopg.pq import Format
+from psycopg.types import TypeInfo
+from numpy import ndarray
+from ..utils.serializer import from_db_str, to_db_str
+
+__all__ = ["register_vector"]
+
+
+class VectorDumper(Dumper):
+    format = Format.TEXT
+
+    def dump(self, obj):
+        return to_db_str(obj).encode("utf8")
+
+
+class VectorLoader(Loader):
+    format = Format.TEXT
+
+    def load(self, data):
+        if isinstance(data, memoryview):
+            data = bytes(data)
+        return from_db_str(data.decode("utf8"))
+
+
+def register_vector(context: Connection):
+    info = TypeInfo.fetch(context, "vector")
```

Comment:
Are we using "vector" or "vectors"?

Reply:
The extension name is "vectors" and the type name is "vector"